specs/NNNN-template/README.md

```diff
@@ -4,12 +4,6 @@
 * **Author(s):** Name (mail), ...
 * **Status:** draft | implementable | implemented | rejected | withdrawn | replaced
 
-<!--
-**Note:** Any PRs to move a VEP to `implementable`, or significant changes once
-it is marked `implementable`, must be approved by at least 2 maintainers .
-
--->
-
```

Comment:
Why editing the VEP template itself?

Reply:
This comment was redundant because we always need 2 approvers now so this doesn't need to be explicit.

Reply:
please create a separate PR for such changes that are not related to the frontend change?

Reply:
Cool